_release/bat/README.md

```diff
@@ -20,10 +20,10 @@ existing test case runner, test-cases.
 
 ## Set up
 
-The BAT tests require a running ciao cluster to execute.  This can be a
+The BAT tests should run from the controller node of a ciao cluster.  This can be a
```

Comment:
Looks like @albertomurillo 's f663ef4 modification addresses this additional paragraph request so I'm going to merge.